### PR TITLE
Add Termination Category

### PR DESCRIPTION
**Pull request by WilliameRocha**
Add termination category to depthk wrapper

### CODE REVIEW

File: benchexec/tools/depthk.py [hunk 1]
```diff
@@ -36,17 +40,16 @@ class Tool(benchexec.tools.template.BaseTool):
...
         executableDir = os.path.dirname(executable)
         return executableDir
 
     def version(self, executable):
-        version = subprocess.Popen([executable, '-v'],
-                                   stdout=subprocess.PIPE,
-                                   stderr=subprocess.STDOUT).stdout.readline().decode()
-        return version
+
+        return 'DepthK 3.0'
```
Comment (PhilippWendler) [inline benchexec/tools/depthk.py]: As I have repeatedly said, please do not hard-code version numbers in the tool-info!

File: benchexec/tools/depthk.py [hunk 2]
```diff
@@ -3,12 +3,16 @@
 """
 BenchExec is a framework for reliable benchmarking.
 This file is part of BenchExec.
+
```
Comment (PhilippWendler) [inline benchexec/tools/depthk.py]: Please do not add irrelevant whitespace changes in the same commit as real changes.

File: benchexec/tools/depthk.py [hunk 3]
```diff
@@ -36,17 +40,16 @@ class Tool(benchexec.tools.template.BaseTool):
 
     def executable(self):
 
-        return Util.find_executable('depthk-wrapper.sh')
+        return Util.find_executable('../../svcomp/depthk/depthk-wrapper.sh'
```
Comment (PhilippWendler) [inline benchexec/tools/depthk.py]: Is this change intentional? If yes, please make a separate commit for it and explain why it is necessary. If not, please make sure *before* creating a pull request that it contains exactly the changes you want to make.

**Comment by WilliameRocha** [comment]
Please, reject and close this pull request.

It is wrong.

Im sorry.

Thanks
